templates/partials/introduction/basic.html

```diff
@@ -1,3 +1,5 @@
+{% from "components/panel/_macro.njk" import onsPanel %}
```

Comment:
Think this line can be removed, other than that looks good

Reply:
Removed, thanks!